dencam/mppt.py

```diff
@@ -23,10 +25,14 @@
 
 
 def get_solardisplay_info():
+    '''Reads the solar data and formats it for display'''
     if not os.path.exists('/home/pi/dencam/solar.csv'):
-        return 'File Does Not Exist'
-    with open('/home/pi/dencam/solar.csv', newline='', encoding='utf8') as cf:
-        parsed_csvfile = cf.read()
+        error_msg = "\nSolar information\nnot found\n\nPress second \
+\nbutton and \nrefer to \nset-up instructions"
```

Comment:
Better to format this in a way that keeps string from not being non-indented e.g. 

```python
error_msg = ("\nSolar information\nnot found\n\nPress second "
             + "\\nbutton and \nrefer to \nset-up instructions")
 ```